t/10-jobs-results.t

```diff
@@ -182,4 +182,32 @@ subtest 'saving results' => sub {
     is_deeply(decode_json($details_file->slurp), \%some_test_results, 'overall structure of test results preserved');
 };
 
+subtest 'loading results with missing file in details' => sub {
+    my $some_test_results = {
+        results => [
+            {
+                test_fqn    => 'random:test',
```

Comment:
Needs tidying

Reply:
Looks like i used the wrong version of perltidy.